py-polars/polars/utils/_parse_expr_input.py

```diff
@@ -77,6 +77,7 @@ def parse_as_expression(
     *,
     str_as_lit: bool = False,
     structify: bool = False,
+    wrap: bool = False,
```

Comment:
I specifically chose not to add this - just call `wrap_expr` on the result.

And when you find you need this option, it means you're doing things in Python that should be done in Rust.

I'll leave it alone for now, but really all the logic you implemented here should probably be done in Rust.

Reply:
In that case just need to remove the extraneous docstring entry (odd that one of our many lints didn't catch that)